app/Http/Controllers/ReportsController.php

```diff
@@ -1162,4 +1162,21 @@ protected function getAssetsNotAcceptedYet()
 
         return Asset::unaccepted();
     }
+
+    
+    /**
+    * Displays EOL report.
+    *
+    * @author [A. Rahardianto] [<veenone@gmail.com>]
+    * @since [v6.0.3]
+    * @return View
+    */
+    public function getEOLReport()    
+    {
+        $this->authorize('reports.view');
+        $eol_assets = Asset::orderBy('created_at', 'DESC')->where('purchase_date','<>','')->get();
```

Comment:
Wouldn't `->whereNotNull('purchase_date)` be more straightforward?

Reply:
Ah so there's such syntax 😅, all right I will update it